<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>689</width>
        <height>382</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Motors</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxMotosSync">
          <property name="text">
           <string>Sync</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxMotor1">
          <property name="minimumSize">
           <size>
            <width>667</width>
            <height>163</height>
           </size>
          </property>
          <property name="title">
           <string>Motor 1</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Velocity:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horSliderVel">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>30</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lEditVel">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1" rowspan="3">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>SerialNo:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="labSerialNo1">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Speed:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="labSpeed1">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Current:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="labCurrent1">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2" rowspan="3">
              <widget class="QPushButton" name="pushButtonResetEnc1">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Encoder:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="labEncoder1">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_12">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Voltage:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="labVoltage1">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>92</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>92</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Acceleration:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horSliderAcc">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>30</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lEditAcc">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButtonStop1">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxMotor2">
          <property name="minimumSize">
           <size>
            <width>667</width>
            <height>163</height>
           </size>
          </property>
          <property name="title">
           <string>Motor 2</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Velocity:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horSliderVel2">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>30</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lEditVel2">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1" rowspan="3">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_14">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>SerialNo:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="labSerialNo2">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_16">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Speed:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="labSpeed2">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_18">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Current:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="labCurrent2">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2" rowspan="3">
              <widget class="QPushButton" name="pushButtoResetEnc2">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_20">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Encoder:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="labEncoder2">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_22">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Voltage:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="labVoltage2">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_24">
               <property name="minimumSize">
                <size>
                 <width>92</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>92</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Acceleration:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horSliderAcc2">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>30</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lEditAcc2">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButtonStop2">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>IR Sensors</string>
       </attribute>
       <widget class="QGraphicsView" name="graphicsViewIR">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>400</width>
          <height>400</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>30</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>IR Sensor1:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>30</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>60</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>IR Sensor2:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>60</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>90</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>IR Sensor3:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>90</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>120</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>IR Sensor4:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>120</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>150</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>IR Sensor5:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>150</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>180</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>IR Sensor6:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>180</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>210</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>IR Sensor7:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>210</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>240</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>240</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>IR Sensor8:</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuRobot">
    <property name="title">
     <string>Robot</string>
    </property>
    <addaction name="actionMotors"/>
   </widget>
   <addaction name="menuRobot"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMotors">
   <property name="text">
    <string>Motors</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
